CMakeLists.txt

```diff
@@ -12,6 +12,23 @@ endforeach()
 
 project(mavsdk_superbuild)
 
+# ccache
```

Comment:
Why are we moving that here? Does it affect the superbuild?

Reply:
I was hoping that it would enable ccache for the superbuild dependencies.

Reply:
I don't understand completely how this cache config works in cmake, but it seems to be just about setting a few variables. Those would have to be propagated to the dependencies (I would say that they are not in the current setup)

Reply:
You're right 🤔 .